test/Microsoft.OData.ModelBuilder.Tests/PublicApi/Microsoft.OData.ModelBuilder.PublicApi.bsl

```diff
@@ -1651,7 +1652,6 @@ public class Microsoft.OData.ModelBuilder.Capabilities.V1.AnnotationValuesInQuer
 
 	string TermName  { public virtual get; }
 
-	public Microsoft.OData.ModelBuilder.Capabilities.V1.AnnotationValuesInQuerySupportedConfiguration HasDynamicProperty (string name, object value)
```

Comment:
@xuzhg -- Removing these will be a breaking change. Can we keep them, but mark as obsolete and remove in next major version?